src/MCPClient/lib/clientScripts/archivematicaClamscan.py

```diff
@@ -23,7 +23,6 @@
 
 from __future__ import print_function
 
-import os
```

Comment:
Is this change related?  Or just an extra thing you noticed and chucked in here?